prboom2/src/p_mobj.c

```diff
@@ -2232,22 +2232,25 @@ void P_ThrustMobj(mobj_t * mo, angle_t angle, fixed_t move)
     mo->momy += FixedMul(move, finesine[angle]);
 }
 
-dboolean P_SeekerMissile(mobj_t * actor, angle_t thresh, angle_t turnMax)
+dboolean P_SeekerMissile(mobj_t * actor, angle_t thresh, angle_t turnMax, dboolean usetracer, dboolean seekcenter)
```

Comment:
How about `mobj_t ** tracer, dboolean seekcenter`?

Then the changes are:
`target = *tracer;`

`*tracer = NULL`

Reply:
That's way better -- I need to stop brute forcing these things. ;P